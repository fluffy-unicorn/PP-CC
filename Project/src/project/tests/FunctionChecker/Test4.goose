# Correct use of multiple functions with the same name (arguments must differ)
function int add(int a, int b) {
    return a + b;
}
function int add(int a, int b, int* c) {
    c = a + b;
    return a + b;
}
function void main() {
    int  a = 10;
    int  b = 7;
    int* c;
    print add(a,b);
    print add(a,b,c);
    print c;
}
