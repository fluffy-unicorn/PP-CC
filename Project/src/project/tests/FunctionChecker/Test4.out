Sprockell 0 says 17
Sprockell 0 says 17
Sprockell 0 says 17
